### ISSUE TITLE
Split out ECN "capable" state

### ISSUE THREAD

**Issue by rmarx** [quicwg/qlog#397]
`ECNState` currently has 4 different options.

During IETF 118, it was mentioned by @mirjak that the `capable` option is potentially not fine-grained enough.
I don't know enough about ECN/L4S to know what the change should be here, but the discussion is on YouTube here: https://youtu.be/qbAm_HfLv_c?feature=shared&t=4419

Something for @marten-seemann to follow-up on

**Comment by mirjak** [comment]
I don't think you need more fine-grained options for the capable ECNState but maybe another state to indicate if the sender, that is in the capable state, also marks packets as ECN capable. In the capable state it can mark packets with either Not-ECT, ECT(0), ECT(1). Where ECT(1) is used to indicate L4S support.
